database/query/notifications.sql

```diff
@@ -0,0 +1,4 @@
+-- name: CreateNotification :one
+INSERT INTO notifications (event_type, flight_number, created_at)
+VALUES ($1, $2, NOW())
```

Comment:
Don't need the `created_at`

```suggestion
INSERT INTO notifications (event_type, flight_number)
VALUES ($1, $2)
```